data/org.freedesktop.impl.portal.Screenshot.xml

```diff
@@ -50,6 +50,17 @@
           Hint whether the dialog should offer customization before taking a screenshot.
           Defaults to no.
 
+        * ``type`` (``u``)
```

Comment:
I think `type` is not very descriptive, because many parts of D-Bus and GLib and etc already use "type" for other things. When it comes to screenshots, I feel like `mode` fits better. What do you think?